app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
When I signed up with my information , the app didn't ask me to confirm my email as stated in the project requirements because you didn't add `:confirmable` to your devise , Kindly fix this issue 😄 😄 👍